Sources/WalletConnect/Types/Session/SessionSequence.swift

```diff
@@ -13,8 +13,12 @@ struct SessionSequence: ExpirableSequence {
     let relay: RelayProtocolOptions
     let controller: AgreementPeer
     let participants: Participants
-    var blockchain: SessionType.Blockchain
-    var permissions: SessionPermissions
+
+//    let `self`: Participant
```

Comment:
I think we don't this commented code anymore

Reply:
We are going to flatten participants as well so actually it will become valid